preprod.healdata.org/manifest.json

```diff
@@ -240,7 +240,7 @@
       "activeDeadlineSeconds": 600,
       "container": {
         "name": "job-task",
-        "image": "quay.io/cdis/batch-export:3.0.0",
+        "image": "quay.io/cdis/batch-export:2022.03",
```

Comment:
this one doesn't have the updated SDK version I think

Reply:
this is the difference I see between both versions (https://github.com/uc-cdis/sower-jobs/compare/2022.03..3.0.0):
`RUN pip install gen3==4.6.3`